### PR TITLE
Kev/413_page

### PR DESCRIPTION
**Pull request by junhaow1**
# Pull Request Details

## What issue does this PR address

- DATASET: After BUILD go to the second page. #413

- Link to associated issue: #413 

## Checklist

Complete the check-list below to ensure your branch is ready for PR.

Flutter Style Guide: https://survivor.togaware.com/gnulinux/flutter-style.html

- [x] Screenshots included in linked issue
- [x] Changes adhere to the style and coding guideline
- [x] No confidential information
- [x] No duplicated content
- [x] No lint check errors related to your changes (`make prep` or `flutter analyze lib`)
- [ ] Pre-exisiting lint errors noted: [HERE]
- [ ] Integration test `make qtest.tmp` screenshot included in issue
- [ ] Tested on device:
  - [ ] Linux
  - [x] MacOS
  - [ ] Windows
- [x] Added two reviewers

## Finalising

Once PR discussion is complete and reviewers have approved:

- [ ] Merge dev into the branch
- [ ] Resolve any conflicts
- [ ] Add one line summary into CHANGELOG.md
- [ ] Bump appropriate version number in pubspec.yaml
- [ ] Push to git repository and review
- [ ] Merge PR into dev (gjwgit)

### CODE REVIEW

**Comment by gjwgit** [comment]
Fails on first try of loading a dataset?

```
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: 'package:flutter/src/widgets/scroll_controller.dart': Failed assertion: line 157 pos 12: '_positions.isNotEmpty': ScrollController not attached to any scroll views.
#0      _AssertionError._doThrowNew (dart:core-patch/errors_patch.dart:50:61)
#1      _AssertionError._throwNew (dart:core-patch/errors_patch.dart:40:5)
#2      ScrollController.position (package:flutter/src/widgets/scroll_controller.dart:157:12)
#3      PageController.animateToPage (package:flutter/src/widgets/page_view.dart:191:41)
#4      DatasetPopup.build.<anonymous closure> (package:rattle/features/dataset/popup.dart:169:52)
<asynchronous suspension>
```

**Review by gjwgit — CHANGES_REQUESTED** [review]
Blocked until it is manually tested as working.

- [x] DEMO -> results in above exception and no navigation to ROLES page
- [x] EXPLORE -> SUMMARY -> BUILD navigates to SUMMARY page
- [x] VISUAL -> BUILD navigates to BOX PLOT
- [x] CORRELATION -> BUILD dose not navigate the first time but does on a second BUILD
- [x] MODEL -> These seem not to have been updated for the navigation

File: support/flutter.mk
Comment (gjwgit) [inline support/flutter.mk:1]: Why is this reverting a dev update? It should be INTERACT not PAUSE no. That was updated a little while ago.

**Review by gjwgit — CHANGES_REQUESTED** [review]

File: support/flutter.mk
Comment (junhaow1) [reply support/flutter.mk:1]: This Makefile rule change aimed to fix the Xcode warnings on macOS ('make qtest.tmp command'), now replacing PAUSE with INTERACT.

Comment (gjwgit) [reply support/flutter.mk:1]: Thanks for the explanation. Yes, I think redirecting stderr to /dev/null makes sense.

**Review by gjwgit — APPROVED** [review]

**Pull request merged**